src/idaapi.py

```diff
@@ -16,68 +17,88 @@
 PLUGIN_OK = 1
 SETMENU_APP = 1
 
+# uefi re tool
+PLUGIN_MOD = 1
+PLUGIN_PROC = 64
+PLUGIN_FIX = 128
+BADADDR = hex(4294967295)
+
+action_desc_t = ida_kernwin.action_desc_t()
+get_input_file_path = ida_nalt.get_input_file_path
+ask_str = ida_kernwin.ask_str
+get_imagebase = ida_nalt.get_imagebase
+
 
 def action_desc_t(*args):
```

Comment:
Not sure if thes args come from Ghidra, but I would create a class to wrap and provide methods to manipulate/operate them.

I'm seeing a lot of global functions receiving *args as argument. I would encapsulate all of them in a class. However, if these functions (like this one) are part of the API that you are mimicking, then ignore this comment.